vb-fe/src/core/player/errorState.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { createContext, useContext, useEffect, useState } from "react";
```

Comment:
Is this how your company generally uses context?  I'm wondering if I should give critiques/suggestions of it or if this is actually an established way -- I haven't seen something like this before, it's really interesting